Assets/Scripts/Player/PlayerController.cs

```diff
@@ -16,37 +12,37 @@ public class PlayerController : MonoBehaviour {
         this.floorMask = LayerMask.GetMask ("Floor");
     }
 
-    void Start () {
-        this.playerRigidbody = GetComponent<Rigidbody> ();
+    protected override void Start () {
+        base.Start();
         this.inventory = GetComponent<PlayerInventory> ();
+        baseStats [Stat.Speed] = 5.0f;
```

Comment:
Sorry, stream of consciousness typing = bad!

Right, in short: since the inspector requires raw types or expressly serializable types (of which Dictionary is not) stats could be housed in a serializable tuple (this is the purpose of StatValue) which can then itself be declared as the generic type of a List. In the inspector, you'll then be able to specify the number of stats you want to provide along with their associated values (the sword and tophat show this off right now). But, it would be possible to provide multiple values for the same stat, so at 'Start' time, this List<StatValue> (or whatever) could be converted into a Dictionary<Stat, float> by reading through the values defined via the inspector one by one. If the stat for a stat+value tuple is ever found to have already been placed into the dictionary, that tuple can be ignored.